cs-couchbase/src/main/java/io/cloudslang/content/couchbase/utils/InputsUtil.java

```diff
@@ -34,25 +48,47 @@ public static HttpClientInputs getHttpClientInputs(String username, String passw
                                                        String keystore, String keystorePassword, String connectTimeout,
                                                        String socketTimeout, String useCookies, String keepAlive, String method) {
         HttpClientInputs httpClientInputs = new HttpClientInputs();
-        httpClientInputs.setUsername(username);
-        httpClientInputs.setPassword(password);
-        httpClientInputs.setProxyHost(proxyHost);
-        httpClientInputs.setProxyPort(proxyPort);
-        httpClientInputs.setProxyUsername(proxyUsername);
-        httpClientInputs.setProxyPassword(proxyPassword);
-        httpClientInputs.setTrustAllRoots(trustAllRoots);
-        httpClientInputs.setX509HostnameVerifier(x509HostnameVerifier);
-        httpClientInputs.setTrustKeystore(trustKeystore);
-        httpClientInputs.setTrustPassword(trustPassword);
-        httpClientInputs.setKeystore(keystore);
-        httpClientInputs.setKeystorePassword(keystorePassword);
-        httpClientInputs.setConnectTimeout(connectTimeout);
-        httpClientInputs.setSocketTimeout(socketTimeout);
-        httpClientInputs.setUseCookies(useCookies);
-        httpClientInputs.setKeepAlive(keepAlive);
         httpClientInputs.setMethod(method);
         httpClientInputs.setAuthType(BASIC);
         httpClientInputs.setQueryParamsAreURLEncoded(Boolean.FALSE.toString());
```

Comment:
you could use io.cloudslang.content.constants.BooleanValues.FALSE (and static import it)